src/error_messages/warning_messages.h

```diff
@@ -66,7 +66,8 @@
 #define FIM_DATABASE_NODES_COUNT_FAIL           "(6948): Unable to get the number of entries in database."
 #define FIM_CJSON_ERROR_CREATE_ITEM             "(6949): Cannot create a cJSON item"
 #define FIM_REGISTRY_ACC_SID                    "(6950): Error in LookupAccountSid getting %s. (%ld): %s"
-
+#define FIM_WHODATA_ERROR_CHECKING_POL          "(6951): Unable to check the necessary policies for whodata: %s (%lu)."
+#define FIM_WHODATA_POLICY_CHANGE               "(6952): Audit policy change detected. Switching directories to realtime."
```

Comment:
This log message is used for both methods (state checker and event detection). It would be useful to distinguish both cases to know how the change was detected.

Reply:
Done